releasenotes/notes/update-sparse-observable-error-2bb4b9e678675eee.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    Binary arithmetic operations on :class:`.SparseObservable` have a slightly updated
+    error message, if the two observables in the operation have incompatible numberss of qubits.
+    Previously, the resulting ``ValueError`` contained the message
+    ``incompatible numbers of qubits: A and B``, which now is
+    ``mismatched numbers of qubits: A, B``.
```

Comment:
This is good thinking, but I don't think we really need to consider this an interface break.

Reply:
Ah even better -- I was just being double safe 😉